djangocms_versioning/models.py

```diff
@@ -21,6 +21,13 @@
     emit_content_change = None
 
 
+def allow_deleting_versions(collector, field, sub_objs, using):
+    if getattr(settings, "DJANGOCMS_VERSIONING_ALLOW_DELETING_VERSIONS", False):
```

Comment:
This setting should be defined in conf.py: https://github.com/django-cms/djangocms-versioning/blob/master/djangocms_versioning/conf.py

This is probably a function that should live in helpers.py, it can then be reused again in the package.